### ISSUE TITLE
Doesn't save cookies

### ISSUE THREAD

**Issue by ElCunq** [cylonid/NativeAlphaForAndroid#100]
Hey I'm using ur app and it's freaking awesome. Great job man. But I'm having an issue. App deletes cookies after closing the shortcut. I oppened accept cookies in shortcut settings and app settings. But it didn't solve. Can you help about that?

**Comment by cylonid** [comment]
This sounds strange. Which website are you referring to?

**Comment by PeterHevesi** [comment]
Yes, I can confirm, cookies are not saving, so GDPR prompts appear again and again...
It works like this on every single website I have installed as app...

**Comment by Hate9** [comment]
I had this issue, and it was fixed by changing the "Accept third-party cookies" setting.
